pyIndego/indego_client.py

```diff
@@ -137,6 +137,10 @@ def update_network(self):
         """Update network."""
         self._update_network(self.get(f"alms/{self._serial}/network"))
 
+    def update_config(self):
```

Comment:
Please implement it the same way but with await before self.get in indego_async_client